manualTests/src/main/x/EchoTest.x

```diff
@@ -0,0 +1,15 @@
+/**
+ * This test is invoked by the build. It can also be invoked manually using the following command:
+ *
+ *     ./gradlew :manualTests:runXtc
+ */
+module EchoTest {
+    void run(String[] args = []) {
+        @Inject Console console;
```

Comment:
Nitpick. This comment will get stale fast, and is still kind of weird. _Every_ test can be invoked by the build, and with runXtc with different configurations and properties. If you have this comment here, you might as well copy/paste it to every single test in the ManualTests project src source.